src/middlewared/middlewared/plugins/failover_/boot.py

```diff
@@ -0,0 +1,58 @@
+from middlewared.schema import accepts, Bool, Dict, returns, Str
```

Comment:
We need to add the copyright header to this file. Please see `failover_/disabled_reasons.py` as an example. Can just copy and paste it.